caesars-palace/blocks/fragment/fragment.css

```diff
@@ -0,0 +1 @@
+/** Placeholder CSS **/
```

Comment:
I think this file can be removed if it isn't needed.

Reply:
Indeed, removing it. I wasn't sure there wouldn't be complaints about the missing file, but seems the Franklin lib ans other scripts are happy not having it too.